docs/snapshot_restore.md

```diff
@@ -93,6 +93,17 @@ start using it.
 At this point, the VM is fully restored and is identical to the VM which was
 snapshot earlier.
 
+## Restore a VM with new Net FDs
+For a VM created with FDs explicitly passed to NetConfig, the snapshot `config.json` will contain invalid fds, for example `"fds":[-1,-1,-1,-1]`. New net FDs have to be passed to restore the VM properly with the syntax `net_fds=[<net_id>@[<fd1>,<fd2>,..],..]`
+```bash
+# First terminal
+./cloud-hypervisor --api-socket /tmp/cloud-hypervisor.sock
+
+# Second terminal
+./ch-remote --api-socket=/tmp/cloud-hypervisor.sock restore source_url=file:///home/foo/snapshot net_fds=[net1@[23,24],net2@[25,26]]
+```
+In the above mentioned example, FDs 23, 24 will be associated to NetConfig with id 'net1' and FDS 25, 26 to NetConfig with id 'net2'.
+
```

Comment:
It is normally better to avoid implementation details and focus on providing concise instructions from the document. A better way to explain it may be:

```
## Restore a VM with Net Devices Backed by FDs

For such case, a set of valid FDs need to be provided along with the VM
restore command in the following syntax:

...

In the example above, the net device with id `net1` will be backed by
FDs '23' and '24' and the net device with id `net2` will be backed by
FDs '25' and '26' from the restored VM.
```

Please also wrap the line at 80 (except code sections), and add white lines before and after the code section.

Reply:
Updated.